src/app/templates/activity/OperStack.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { memo, useMemo, useState } from 'react';
+
+import classNames from 'clsx';
+
+import { OP_STACK_PREVIEW_SIZE } from 'app/defaults';
+import { ReactComponent as ChevronRightIcon } from 'app/icons/chevron-right.svg';
+import { ReactComponent as ChevronUpIcon } from 'app/icons/chevron-up.svg';
+import { T } from 'lib/i18n/react';
+import { OperStackItemInterface } from 'lib/temple/activity-new/types';
+
+import { OperStackItem } from './OperStackItem';
+
+type Props = {
+  opStack: OperStackItemInterface[];
+  className?: string;
+};
+
+export const OperStack = memo<Props>(({ opStack, className }) => {
```

Comment:
still `oper`

Reply:
we have got `op` shortening for `operations` already.  
I think it's better not to use shortenings or at least use the same naming

Reply:
the same in ither places